manageiq-ui-classic.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.executables   = s.files.grep(%r{^exe/}) { |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_dependency "rails", "~> 5.0.0", ">= 5.0.0.1"
+  s.add_dependency "rails", ">= 5.0.0.1", "< 5.1"
```

Comment:
Apologies. So by 'switch this line (in master) to better match what is in the core repo' you mean switch what is in the core repo to better match this.

I don't think it's necessary, but feel free. I think the difference here is an application dependency vs a gem dependency. The application can specify whatever it wants (because nothing depends on it) but the gem here needs to be more open (because it sets a restriction on things that depend on it).